src/mutable-source-code.ts

```diff
@@ -1,7 +1,7 @@
-import ts = require('typescript');
+import * as ts from 'typescript';
```

Comment:
The existing code was using both. You have 5 files that are using the `import * as ts` version vs 6 using `import ts = require()`. I only intended to standardize them rather than using both. Since it was roughly split 50/50, I chose the syntax I like better. Feel free to change them all to the other version if you prefer.